app/src/main/java/Activities/ViewRecordActivity.java

```diff
@@ -69,6 +73,10 @@ protected void onCreate(Bundle savedInstanceState) {
             e2.printStackTrace();
         }
 
+        this.currentgeo = new GeoLocationController().getGeoLocation(ViewRecordActivity.this,this.recordUUID);
```

Comment:
I think it is because of the geolocation text view, I can fix that, I will make new commits to fix in my new pull request. the aggregate map view pull request should cover everything.

Reply:
Ok, I'll close this one then